shared/native/notification-listeners.native.js

```diff
@@ -9,5 +10,18 @@ export default function(dispatch: Dispatch, getState: () => Object, notify: any)
   const fromShared = shared(dispatch, getState, notify)
   return {
     ...fromShared,
+    'keybase.1.NotifyBadges.badgeState': ({badgeState}) => {
+      const sharedBadgeState = fromShared['keybase.1.NotifyBadges.badgeState']
+      sharedBadgeState({badgeState})
+
+      const count = (badgeState.conversations || []).reduce((total, c) => total + c.UnreadMessages, 0)
+
+      if (count !== -1) {
```

Comment:
actually this was a leftover when i was trying to send this value from the fromShared version (but then that causes a bunch of flow issues and isn't worth it) so this is basically impossible and serves no purpose so i'm removing.